src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,21 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
+    QRegExp rx("<(.+)>");
+    rx.setMinimal(true);
+    QString result = savePathTpl;
+    int pos = 0;
+    while ((pos = rx.indexIn(result, pos)) != -1) {
+        QString value = article->getValue(rx.cap(1));
+        result.replace(rx.cap(0), value);
```

Comment:
Doesn't cap(0) have all matching words? So wouldn't be better to use the pos and size of cap(1) to replace it?
PS: I could be wrong on what I say about cap(0) since I don't understand regex very much. So please give me a simple code example/explanation if your code is actually correct.